components/datadog/dockeragentparams/params.go

```diff
@@ -32,7 +31,7 @@ import (
 // [Functional options pattern]: https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis
 
 type Params struct {
-	// FullImagePath is the full path of the docker agent image to use.
+	// AgentFullImagePath is the full path of the docker agent image to use.
 	// It has priority over ImageTag and Repository.
 	FullImagePath string
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should you change the param name too ?